pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -366,6 +366,7 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 	}
 
 	addControllerManagerNetworkArgs(clusterConfig.ControllerManager.ExtraArgs, cluster.ClusterNetwork)
+	addControlPlaneComponentsAdditionalArgs(cluster, clusterConfig)
```

Comment:
This might be too early, I have to double check, but later functions might override some flags and feature gates.